chain/dependencies.go

```diff
@@ -244,6 +246,7 @@ type Action interface {
 
 type Auth interface {
 	Object
+	Marshaler
```

Comment:
`Marshaller` should have two `l`s.

Reply:
Thanks, will use "Marshal" according to Go conventions